src/navigations/root-stack.js

```diff
@@ -187,7 +187,7 @@ const AuthenticatedNavigator = () => {
         />
         <AuthenticatedStack.Screen
           name="OtherProfile"
-          component={withKeyboardWrapper(OtherProfile)}
+          component={withSafeAreaView(withKeyboardWrapper(OtherProfile))}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change wraps the `OtherProfile` component with `withSafeAreaView`. This is a good practice as it ensures that the component will be displayed correctly on devices with notches or rounded corners. However, please ensure that the `withSafeAreaView` higher-order component does not interfere with the styles and layout of the `OtherProfile` component.
```diff
-          component={withKeyboardWrapper(OtherProfile)}
+          component={withSafeAreaView(withKeyboardWrapper(OtherProfile))}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->